pkg/agent/openflow/client.go

```diff
@@ -1190,6 +1204,15 @@ func (c *client) UninstallMulticastFlows(multicastIP net.IP) error {
 	return c.deleteFlows(c.featureMulticast.cachedFlows, cacheKey)
 }
 
+func (c *client) InstallMulticastToRemoteFlows(groupID binding.GroupIDType) error {
+	firstMulticastTable := c.pipelines[pipelineMulticast].GetFirstTable()
+	flows := c.featureMulticast.multicastToRemoteFlows(groupID, firstMulticastTable)
+	cacheKey := "multicast-encap"
```

Comment:
It seems all the cacheKey in `client.go` use underscore. Maybe `multicast_encap`?

Reply:
updated.